packages/happy-dom/src/nodes/node/Node.ts

```diff
@@ -351,6 +359,60 @@ export default class Node extends EventTarget implements INode {
 		return newNode;
 	}
 
+	/**
+	 * Inserts a node to the given position.
+	 *
+	 * @param position Position to insert element.
+	 * @param element Node to insert.
+	 * @return Inserted node or null if couldn't insert.
+	 */
+	public insertAdjacentElement(position: TInsertAdjacentPositions, element: INode): INode | null {
```

Comment:
insertAdjacentElement(), insertAdjacentHTML(), insertAdjacentText() is part of the IElement interface and should be implemented in the Element class.